### PR TITLE
doc: (#8906) UI update what's new entries for 0.13.0

### LINKED ISSUE #8906: UI : update what's new entries for 0.13.0
Update what's new entries for 0.13.0

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #8906
### Describe your changes :

I worked on issue #8906 

#
### Type of change :

- [x] Documentation

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Review by ShaileshParmar11 — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts [hunk 1]
```diff
@@ -13,9 +13,9 @@
 
 /* eslint-disable max-len */
 
-export const LATEST_VERSION_ID = 9;
+export const LATEST_VERSION_ID = 10;
 
-export const COOKIE_VERSION = 'VERSION_0_12_1'; // To be changed with each release.
+export const COOKIE_VERSION = 'VERSION_0_13_0'; // To be changed with each release.
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts:18]: Exactly where we are using this?

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts:18]: We are using this in 2 places,
1. to keep track of the recently viewed and recently searched data for the current version
2. for what's new modal, if a user is upgrading from the old version to the new version then on the first login we show the modal based on this cookie.

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts [hunk 2]
```diff
@@ -548,4 +548,61 @@ export const WHATS_NEW = [
...
+-   Upload a CSV with thousands of terms in one go.
+-   Define the **Owners and Reviewers** during bulk upload.
+-   Owners and Reviewers will be propagated to every glossary term.
+-   Users can **Preview** before uploading to ensure that everything looks right.
+-   Support a **Review Workflow** for new term suggestions.
+-   Suggested terms will be saved as Draft terms.
+-   Admins can review and approve these draft terms.
+-   Once approved, these terms can be used for tagging and searching.
+-   A glossary term can be moved to a different hierarchy within a glossary or across a glossary.
+-   While moving a term, even the child terms will be moved along with the parent term.`,
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts:577]: I feel glossary is yet to done and not part of 0.13.0
cc: @harshach

Comment (harshach) [reply openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts:577]: yes lets remove that

**Review by harshach — APPROVED** [review]

**Pull request merged**